languages/python/exercises/concept/tuples/.meta/example.py

```diff
@@ -0,0 +1,21 @@
+
+def get_coordinate(record):
+    return record[1]
+
+def convert_coordinate(coordinate):
+    # alt return coordinate[0], coordinate[1]
+    return tuple(coordinate)
+
+def compare_records(azara_record, rui_record):
+    return convert_coordinate(azara_record[1]) in rui_record
+
+def create_record(azara_record, rui_record):
+    if compare_records(azara_record, rui_record):
+        return azara_record + rui_record
+    else:
+        return "not a match"
+
+def clean_up(combined_record_group):
+    for item in combined_record_group:
+        clean_record = item[0], item[2], item[3], item[4]
+        print(clean_record)
```

Comment:
It does indeed print to the console.  Not at all ideal, but I'm struggling here because tuples are immutable.  The point of this last task was to iterate through a tuple and do something with the data (in this case 'dropping' an element from each 'record' within the outer tuple) -- but making a _new_ tuple of tuples iteratively via the **+** operator means that the data gets unpacked -- so the nested tuples become un-nested.  😞 

Alternatively, the inner tuples could be added into a `list` that is then converted --  but lists are neither required / taught for this exercise. The input could be changed to something weird like double-wrapped tuples (three layers of nesting as opposed to two) so that the **+** doesn't completely unpack them - but that seems very contrived.  Maybe if this required a tuple of each team members records as input with the output being a combined tuple....

Hum.  Suggestions here?  I will sleep on it and try to think of an alternate first thing in the morning.  😁

Reply:
> The point of this last task was to iterate through a tuple and do something with the data

Maybe outputting the combined records in a new string format? Would something like that work?

Reply:
> Maybe outputting the combined records in a new string format? Would something like that work?

This is similar to what we do in the `tournament` exercise, and would be preferable here.

Reply:
Yes!  Making a string and compiling a 'report'  seems like a **much** better solution - and `strings` are required knowledge for this exercise, so something with `.format()` or `f-strings` in a report would work well.  I'll take a look at `tournament` and how the instructions read there, and change this task accordingly.  _**thank you both**_ for getting me un-stuck!  😀

Reply:
Our pleasure!